.github/workflows/gradle.yml

```diff
@@ -22,25 +22,26 @@ on:
 jobs:
   changes:
     runs-on: ubuntu-latest
+    outputs:
+      java: ${{ steps.changes.outputs.java }}
+      python_cdk: ${{ steps.changes.outputs.python_cdk }}
```

Comment:
Introduce python cdk change detection as we'd want to add a job to run python cdk tests